libbeat/docs/processors-using.asciidoc

```diff
@@ -32,6 +32,26 @@ condition is present, then the action is executed only if the condition is
 fulfilled. If no condition is passed, then the action is always executed.
 * `<parameters>` is the list of parameters to pass to the processor.
 
+ifeval::["{beatname_lc}"=="filebeat"]
```

Comment:
@andrewkroh I'll need to add some conditional coding to this topic because we probably need examples for each beat. Incidentally, when I was looking into what to add here, I looked at the reference.yml files for each beat to figure out where  processors are valid. It's not clear that they are supported at the module level for auditbeat, metricbeat, and winlogbeat.

Reply:
I agree that the reference configs are not clear about where processors can be used. Here's where they can be used. The list is derived from this [search](https://github.com/elastic/beats/search?l=Go&q=processors.PluginConfig).

- auditbeat.modules
- filebeat.inputs
- heartbeat.monitors
- metricbeat.modules
- packetbeat.flows
- packetbeat.protocols